goLF/new.go

```diff
@@ -1,19 +1,24 @@
 package goLF
 
 import (
+	"strconv"
 	"database/sql"
+	"github.com/elastic/go-elasticsearch/v8"
+
 	"github.com/LetsFocus/goLF/configs"
 	"github.com/LetsFocus/goLF/database"
+	"github.com/LetsFocus/goLF/elasticstack"
 	"github.com/LetsFocus/goLF/logger"
 )
 
 type GoLF struct {
 	Conn   *sql.DB
```

Comment:
lets create a struct Database and inside it lets keep all the DBs inside that struct